css/style.css

```diff
@@ -0,0 +1,106 @@
+.checkerboard{
+	position: relative;
+	margin: 0 auto;
+	padding: 30px;
+	height: 560px;
+	width: 560px;
+	background-color: #E1D7C9;
+}
+
+.checkerboard__square{
+	position: absolute;
+	height: 70px;
+	width: 70px;
+}
+
+.row1:nth-child(odd),
+.row2:nth-child(even), 
+.row3:nth-child(odd),
+.row4:nth-child(even), 
+.row5:nth-child(odd),
+.row6:nth-child(even),
+.row7:nth-child(odd),
+.row8:nth-child(even){
+	background-color: #1D201F;
+}
+
+.row1{
```

Comment:
if you use BEM you  than it is not correct
you should wright smth like
table__row--first